snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -428,8 +423,16 @@
         var snippet_id = ABOUTHOME_SHOWN_SNIPPET.id;
         var blockSnippet = function (event) {
             event.preventDefault();
+            // We want to stop other click events.
+            // This element is being removed, so another
+            // click might cause parentNode reference errors.
+            // See trackLinkClicks()
+            event.stopPropagation();
```

Comment:
`trackLinkClicks` function is called whenever `topSection` is clicked, but its `useCapture` is `true` which was causing issues when firing the event after the snippet was removed.

I covered it here, but also just in case `trackLinkClicks` was also updated to handle the case where the element was removed from the DOM.